nexus/nexus-deployment-template.yml

```diff
@@ -52,6 +52,19 @@ objects:
       name: "${NAME}"
     strategy:
       type: Recreate
+      recreateParams:
+        post:
+          failurePolicy: Abort
+          execNewPod:
+            command:
+              - /bin/sh
+              - '-c'
+              - /repos/repos.sh
+            containerName: "${NAME}"
+            volumes:
+              - "${NAME}-cm"
+              - "${NAME}"
```

Comment:
This will become a problem if the post hook pod is scheduled on a different node than the nexus pod as the PVC has access mode `RWO`. Not sure if it is possible to use affinity rules for this or if it would require the PVC to be `RWX`.

Reply:
ok. I'm going to back out this change and leave just the probes. It doesn't look like there is a way to configure the lifecycle pod to run on the same node as the nexus pod without the nexus pod being tied to a specific node which doesn't seem right for a generic template. I'll have to find another way (docker build likely).

The volume mount was there so that it could pick up a newly generated password that happens when nexus is first deployed in versions 3.17+

Reply:
@pabrahamsson removed the post hook